blank_project/assembly/json/decoder.ts

```diff
@@ -0,0 +1,332 @@
+declare function logStr(str: string): void;
```

Comment:
is this supposed to be here? I thought it was a dependency brought from ts-runtime?

Reply:
it is debugging left over in assemblyscript-json:
https://github.com/nearprotocol/assemblyscript-json/blob/master/assembly/decoder.ts#L1

Long-term prob should be some common ABI for this stuff usable both on NEAR and without NEAR (as parser is standalone project).

Reply:
I meant the whole file. The way shell worked before, it would bring over this file from dependencies without having to have decoder.ts here.

Reply:
Changed so that `new_project` copies it for now (so generated projects have copy, but it's taken from node_modules).

 Gonna fix to resolve at compile-time after figuring proper solution for NEARStudio (looks like it's possible to get one of package managers work in-browser with help of `fs` module implementation after patching out some other stuff).